inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -204,11 +204,25 @@ def match_reference(reference):
 
     config_default = current_app.config['WORKFLOWS_REFERENCE_MATCHER_DEFAULT_CONFIG']
     config_jcap_and_jhep = current_app.config['WORKFLOWS_REFERENCE_MATCHER_JHEP_AND_JCAP_CONFIG']
+    config_data = current_app.config['WORKFLOWS_REFERENCE_MATCHER_DATA_CONFIG']
 
     journal_title = get_value(reference, 'reference.publication_info.journal_title')
-    config = config_jcap_and_jhep if journal_title in ['JCAP', 'JHEP'] else config_default
+    if journal_title in ['JHEP', 'JCAP']:
+        config = config_jcap_and_jhep
+        if get_value(reference, 'reference.publication_info.year'):
+            reference['reference']['publication_info']['year'] = str(reference['reference']['publication_info']['year'])
```

Comment:
this seems very wrong to me. If ES expects a string, either the mapping or the config needs to be changed. The record schema says that [`year` is an integer](https://github.com/inspirehep/inspire-schemas/blob/963237a736db12324136760c06479d41d6abed9c/inspire_schemas/records/elements/reference.yml#L247-L252), if you change this the reference will fail validation.